packages/@webex/plugin-meetings/test/unit/spec/locus-info/index.js

```diff
@@ -1900,5 +1906,113 @@ describe('plugin-meetings', () => {
         );
       });
     });
+
+    describe('#isMeetingActive', () => {
+      it('sends client event correctly for state = inactive', () => {
```

Comment:
nitpick: feels like these tests are all very similar and could be parametrized, but they're also quite short anyway so feel free to ignore this